sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -783,27 +783,34 @@ case class TimeSub(start: Expression, interval: Expression)
 /**
  * Returns the date that is num_months after start_date.
  */
+// scalastyle:off line.size.limit
 @ExpressionDescription(
-  usage = "_FUNC_(start_date, num_months) - Returns the date that is num_months after start_date.",
+  usage = "_FUNC_(start_date, num_months) - Returns the date/timestamp that is num_months after start_date.",
   extended = "> SELECT _FUNC_('2016-08-31', 1);\n '2016-09-30'")
+// scalastyle:on line.size.limit
 case class AddMonths(startDate: Expression, numMonths: Expression)
```

Comment:
as it's not always date type now, should we use a new name instead of `startDate`?